calico_containers/pycalico/netns.py

```diff
@@ -51,6 +52,49 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    If any default route has a metric of 0, increase the metric of 
+    all default routes by 1.
+    :param namespace: The Networking namespace of the container.
+    :return: None. Raises CalledProcessError on error.
+    """
+    class Route(object):
```

Comment:
Will need UTs.  I think you'll find it easier to UT this class if you declare it outside the scope of the function.